model/riscv_fdext_regs.sail

```diff
@@ -317,7 +317,7 @@ function wF_or_X_D(i, data) = {
     assert (regidx_bits(ridx)[0] == bitzero);
     if ridx != zreg then {
       X(ridx)                         = data[31..0];
```

Comment:
Probably should reformat this... (One of the reasons I generally don't like aligning things.)